lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -391,6 +392,62 @@ def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
         end
       end
 
+      # Update the load balancer sticky sessions
+      if sticky_sessions
+        policy_name = "#{actual_elb.name}-sticky-session-policy"
+        policies = elb.client.describe_load_balancer_policies(load_balancer_name: actual_elb.name)
+        current_cookie_hash = cookie_from_policies(policies)
+
+        # first make sure the policy is in place
+        if current_cookie_hash[:attribute_value] != sticky_sessions[:cookie_name]
+          perform_action.call("  updating sticky sessions to #{sticky_sessions[:cookie_name].inspect}") do
+            count = 0
+
+            begin
+              elb.client.create_app_cookie_stickiness_policy(
+                load_balancer_name: actual_elb.name,
+                policy_name: policy_name,
+                cookie_name: sticky_sessions[:cookie_name]
+              )
+            rescue AWS::ELB::Errors::DuplicatePolicyName
+              if count < 1
+                elb.client.delete_load_balancer_policy(
+                  load_balancer_name: actual_elb.name,
+                  policy_name: policy_name
+                )
+
+                count += 1
+                retry
+              else
+                raise
+              end
+            end
+          end
+        end
+
+        # then make sure it's attached to the appropriate listeners
+        elb_description = elb.client.describe_load_balancers(load_balancer_names: [actual_elb.name])
```

Comment:
@tyler-ball I'm picking this up as Josh is out for a bit.  Is the #reload method something that should exists?  When I try this change locally, I get a NoMethod error.